velox/docs/monitoring/metrics.rst

```diff
@@ -43,7 +42,7 @@ number of shuffle requests per second.
 **Histogram**: tracks the distribution of event data point values, such as query
 execution time distribution. The histogram metric divides the entire data range
 into a series of adjacent equal-sized intervals or buckets, and then count how
-many data values fall into each bucket. DEFINE_HISTOGRAM_STAT specifies the data
+many data values fall into each bucket. DEFINE_HISTOGRAM_METRIC specifies the data
```

Comment:
See:  
https://github.com/facebookincubator/velox/blob/25184631781e6dc9f738efdd1a5276746c11dd95/velox/common/base/Counters.cpp#L393

The name of the macro used to register a histogram-type metric is `DEFINE_HISTOGRAM_METRIC`, not `DEFINE_HISTOGRAM_STAT`, this PR fixes it.